src/components/PetAttributes.jsx

```diff
@@ -9,21 +9,43 @@ const PetAttributes = props => {
   const { attributes } = props;
 
   var displayAttributes = [
-    "Species",
-    "Breed",
-    "Color",
+    "Type",
+    "Primary Breed",
+    "Primary Color",
     "Sex",
     "Weight",
     "Age",
-    "Spayed/Neutered",
+    "Altered",
     "Shelter Arrival Date"
   ];
 
+  //TODO: Possibly needs to be added to the service. But for now this is where this lives
+  const ConvertToFriendlyNames = value => {
+    var newValue = "";
+    switch (value) {
+      case "Type":
+        newValue = "Species";
+        break;
+      case "Primary Breed":
+        newValue = "Breed";
+        break;
+      case "Altered":
+        newValue = "Spayed/Neutered";
```

Comment:
can you make this "Spayed / Neutered" this will resolve one of Jason's bugs. Bascially it allows the table to wrap that text, instead of a one liner.

Reply:
done